server/db/migrations/20220922101628_produce.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('produce', (table) => {
+    table.increments('id').primary()
+    table.string('name')
+    table.string('display_name')
+    table.string('type')
```

Comment:
Based on your readme this can only be 2 options, fruit and vegetables. If thats the cause then make it a enum (enu in knex) rather than a string. If you keep it as string then it means anything could go in which could cause issue

```suggestion
    table.enu('type', ['fruit', 'vegetable']).notNullable()
```

Reply:
Great suggestion!